prestodb/presto
# ignore non-existent partition location

hive and impala simply ignore partitions with non-existent hdfs location,
but presto throws an exception in this case.

test scenario:
1. create a partition
2. hdfs dfs -rmdir "location of created partition"
3. run a select references that partition
4. you'll see a "Partition location does not exist" exception.
